src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const array = [...document.querySelectorAll('li')];
+
+const toNum = (el) => {
+  return parseFloat(el.replace(/[$,]/g, ''));
+};
+
+const sortList = (list) => {
+  list.sort((a1, a2) => toNum(a2.dataset.salary) - toNum(a1.dataset.salary));
+
+  list.forEach((element) => {
+    document.querySelector('ul').insertAdjacentElement('beforeend', element);
+  });
+};
+
+const getEmployees = (list) => {
+  const result = [];
+
+  list.forEach((element) => {
+    result[result.length] = {
+      name: element.textContent,
+      posititon: element.dataset.position,
```

Comment:
There is a typo in the property name 'posititon'. It should be 'position'. This could cause issues when trying to access this property later.